docs/sources/configure-server/about-server-api.md

```diff
@@ -0,0 +1,194 @@
+---
+description: Learn about the Pyrocope server API
+menuTitle: Server API Overview
+title: Pyroscope Server HTTP API Reference
+weight: 20
+---
+
+# Pyroscope Server HTTP API Reference
+
+Pyroscope server exposes an HTTP API that can be used to query profiling data and to ingest profiling data from other sources.
+
+## Authentication
+TODO - how does authentication work with new Grafana cloud stuff?
+
+## Ingestion
+
+Currently there's just one endpoint: `POST /ingest`. It takes profile data in request body and metadata as query params.
+
+It takes multiple query parameters:
+
+| Name               | Description                             | Notes                           |
+|:-------------------|:----------------------------------------|:--------------------------------|
+| `name`             | application name                        | required |
+| `from`             | UNIX time of when the profiling started | required                        |
+| `until`            | UNIX time of when the profiling stopped | required                        |
+| `format`           | format of the profiling data            | optional (default is `folded`)  |
+| `sampleRate`       | sample rate used in Hz                  | optional (default is 100 Hz)    |
+| `spyName`          | name of the spy used                    | optional                        |
+| `units`            | name of the profiling data unit         | optional (default is `samples`  |
+| `aggregrationType` | type of aggregration to merge profiles  | optional (default is `sum`)     |
```

Comment:
We probably want to review the list of params, and remove some of them:
 - `name` – as Ryan mentioned, we need to make sure the new series labels format is supported;
 - `spyName` – could be part of series labels;
 - `units` - should be part of the profile type (a label);
 - `aggegationType` – unused.